### DISCUSSION TITLE
Proxy Connection

### DISCUSSION THREAD

**Discussion by cjlynch278** [mkleehammer/pyodbc#1030] [Q&A] [answered]
Is there a way we can connect to a sql server behind a proxy? Any help/advice would be appreciated. I've found a couple questions about it on stackoverflow but no answers:

- [59680113](https://stackoverflow.com/questions/59680113/how-to-use-python-or-pyodbc-module-with-proxy-settings)
- [69793411](https://stackoverflow.com/questions/69793411/access-database-behind-proxy-using-pyodbc-sqlalchemy)

**Comment by v-chojas** [comment, marked as answer]
pyODBC is definitely the wrong layer to be asking this question on, but even putting that aside, you'd still need to be far more specific about your environment's details. There are different types of proxies and how you interact with them as well as what's possible will differ too. I recommend you do some reading on that and "proxification" software in general.
